test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol~Stashed changes

```diff
@@ -0,0 +1,16 @@
+contract C {
```

Comment:
This file seems to be included by accident?

Reply:
Thanks for pointing that out! Removed it in the latest push.